hhs_oauth_server/settings/base.py

```diff
@@ -138,8 +138,7 @@
 
 ALLOWED_HOSTS = env("DJANGO_ALLOWED_HOSTS", ["*", socket.gethostname()])
 
-# DEBUG = env("DEBUG", True)
-DEBUG = True
+DEBUG = env("DEBUG", True)
```

Comment:
@JFU-GIT I think this original code should be changed to the following to handle any case where the ENV variable might not be set. It should default to `False` always.

`DEBUG = env("DEBUG", False)`